hack/jenkins/verify.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Runs the unit and integration tests, production JUnit-style XML test reports
+# in ${WORKSPACE}/_artifacts.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
```

Comment:
I basically cargo-culted this file from the other scripts in hack/jenkins. All of them set xtrace, though maybe we should change that?

I did notice that my comment on L17-18 was wrong, so that's been fixed now at least.